document-generation/utils/dow.ts

```diff
@@ -0,0 +1,1113 @@
+import {
```

Comment:
Yikes. Intense.

Reply:
@jasonburkert-ccpo this file will definitely need some refactoring along with the addition of tests. We should create a tech debt ticket to be capture that.